packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Card } from '../cards/Card';
+import { ChecklistStep } from './ChecklistStep';
+import { ChecklistCardProps } from '../../lib/checklist';
+import { useChecklist } from '../../hooks/useChecklist';
+import { ButtonSize } from '../buttons/Button';
+import CloseButton from '../CloseButton';
+import { RankConfetti } from '../../svg/RankConfetti';
+
+const ChecklistCard = ({
+  className,
+  title,
+  description,
+  steps,
+  onRequestClose,
+}: ChecklistCardProps): ReactElement => {
+  const { isDone, openStep, onToggleStep, activeStep } = useChecklist({
+    steps,
+  });
+
+  return (
+    <div className={className}>
+      <Card className="p-0 bg-transparent rounded-14 !border-theme-color-cabbage hover:!border-theme-color-cabbage w-[340px]">
+        <div className="overflow-hidden relative p-4 bg-gradient-to-t from-cabbage-90 to-cabbage-50 rounded-t-12">
```

Comment:
Not sure we should be using `from-cabbage-90 to-cabbage-50`. We need to use the one defined in our config, we might just have to find the correct one. I understand it is a pain for now, but hopefully we can create a documentation mapping this.

Reply:
I think for this it's fine it's a exclusion that's the same render for both light/dark modes 😅
It's a bit tricky but we used it before like this. (Correct me if i'm wrong though)

Reply:
Ah. On the banner. We might probably need to come up with some sort of system for this in the future. Just to be consistent and inline. In that case, kindly disregards this one 🚢

Reply:
I see I posted comment late without seeing your comments. 

Ok, going with this, tnx for checking 🍻

Reply:
Agree with Lee though, @capJavert can you add a backlog ticket just so we have it :)

Reply:
For sure!

Reply:
https://dailydotdev.atlassian.net/browse/WT-1316